pkg/cluster/internal/create/actions/kubeadmjoin/join.go

```diff
@@ -142,7 +188,8 @@ func runKubeadmJoinControlPlane(ec *execContext, configNode *NodeReplica) error
 		// preflight errors are expected, in particular for swap being enabled
 		// TODO(bentheelder): limit the set of acceptable errors
 		"--ignore-preflight-errors=all",
-		kubeadmVerbosityFlag,
+		// increase verbosity for debug
```

Comment:
Do we need to be consistent with other comments?   e.g. `for debugging`

Reply:
🤷‍♂️ -- we should make this configurable with an advanced option in v1alpha3 probably.